Edits on OSS models and addition of Pulumi terms & command equivalents (#3286)
pulumi/pulumi-hugo: themes/default/content/docs/concepts/vs/terraform/_index.md

    }
</style>

> Choosing the right infrastructure as code platform is critical. We want you to have the best information possible to pick the solution that fits your needs.

Reviewer: in the What is TF section 2 para below, we hyperlink infrastructure as code. Can we move that hyperlink to this first instance instead?

Author: yup

> Pulumi and Terraform are both infrastructure as code technologies. Pulumi is open source and offers the flexibility to use any programming language for managing infrastructure. This makes it widely accessible to developers and devops engineers from any background. Terraform requires the use of a proprietary domain-specific language (DSL) and is not open source. In this comprehensive guide, we'll explore the key differences, similarities, and real-world scenarios, mapping Terraform concepts to Pulumi to help you choose the right infrastructure as code platform.

Reviewer 2: Obviously this info's super relevant, but it feels a bit "wedged into the middle" of this introductory paragraph to me. Seems like it belongs in the key differences paragraph instead.

```suggestion
Pulumi and Terraform are both infrastructure as code technologies. In this comprehensive guide, we'll explore their key differences, similarities, and real-world scenarios, mapping Terraform concepts to Pulumi to help you choose the right infrastructure as code platform.
```

If we did want to keep it here, we might just call it out briefly as a "for example", like "Pulumi and Terraform are alike in many ways, but different in many others. Pulumi, for example, is open-source, whereas Terraform is not. In this comprehensive guide, we'll cover these and other key differences in depth..." ... or something. 😄

Getting started with Pulumi is easy. If you have experience with Terraform and already have HCL, you can convert it to Pulumi. Follow our comprehensive guides in our [Migration Hub](/blog/migration-hub/) to use our [self-service migration tools](/docs/using-pulumi/adopting-pulumi/migrating-to-pulumi/from-terraform/) or work with our [Expert Services teams](/contact?form=tf-migration) that can help you with migration and training. As part of Pulumi Enterprise and Business Critical Editions, [Terraform workspace migrations](/blog/tf-migration-offer) are bundled as part of the onboarding process.

> If you would like to see how other Pulumi customers have migrated from Terraform to Pulumi, see the [Atlassian case study](/case-studies/atlassian/) and the [Green Park Sports case study](/case-studies/greenpark-sports/). If you would like to deploy a simple program, follow our Get Started guide:

Reviewer: There is a lot of white space before the Get Started with Pulumi section.

Transformations, which are unique to Pulumi, allow you to programmatically set or override the input properties of resources belonging to a particular collection, such as the child resources of a Pulumi component or even all of the resources belonging to a stack. Transformations make it easy to apply consistent settings across your infrastructure without having to manipulate the properties of individual resources. To learn more, see [Transformations](/docs/concepts/options/transformations/) in the Resource documentation.

> ### Terminology and Commands

Reviewer: I am wondering if we should just put the chart from that terminology page directly into this page and hide it in an accordion.

Reviewer: This is pretty important stuff but its somehow hidden. maybe we need to link to it from above?

Transformations, which are unique to Pulumi, allow you to programmatically set or override the input properties of resources belonging to a particular collection, such as the child resources of a Pulumi component or even all of the resources belonging to a stack. Transformations make it easy to apply consistent settings across your infrastructure without having to manipulate the properties of individual resources. To learn more, see [Transformations](/docs/concepts/options/transformations/) in the Resource documentation.

### Terminology and Commands

> If you’re familiar with Terraform, learning Pulumi terminology and commands is simple. Many of the existing Terraform vocabulary and commands that you already know have direct equivalents in Pulumi. To learn more, see [Pulumi terms & command equivalents](/docs/concepts/vs/terraform/terminology/) in the Resource documentation to find a list of common Terraform terms and CLI commands along with their Pulumi equivalents.

Reviewer 2: Minor nit, but since these don't live in the resource documentation (which is more what registry API docs are):

```suggestion
If you’re familiar with Terraform, learning Pulumi terminology and commands is simple. Many of the existing Terraform vocabulary and commands that you already know have direct equivalents in Pulumi. To learn more, see [Pulumi terms & command equivalents](/docs/concepts/vs/terraform/terminology/) for a list of common Terraform terms and CLI commands along with their Pulumi equivalents.
```
